### DISCUSSION TITLE
Processor name for gen_ens_prod

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1258] [Configuration] [answered]
In the new MET/METpus release (version 4.1.0-beta3), a new tool "gen_ens_prod" was
added, I would like to know its name for PROCESS_LIST in METplus conf file

Binbin

**Comment by georgemccabe** [comment, marked as answer]
Hi @BinbinZhou-NOAA,

While the MET tool gen_ens_prod was added in the MET 10.1.0-beta3 release, the METplus wrapper was added after this release. The wrapper will be available in the METplus 4.1.0-beta4 release which is scheduled for this Monday.

There is an example use case in the parm directory that runs this tool here: https://github.com/dtcenter/METplus/blob/7094e0a613d354d894119900327837b6da143644/parm/use_cases/met_tool_wrapper/GenEnsProd/GenEnsProd.conf#L3

Typically the name of the process is formatted with no underscores or dashes and the first letter of each word capitalized, i.e. "GenEnsProd." However, the logic that parses the PROCESS_LIST values is smart enough to handle many different variations of a tool name, so setting "gen_ens_prod", "GenEnsProd", "Gen-Ens-Prod", etc. will work.

**Reply by BinbinZhou-NOAA** [reply]
George,

  Thanks!

Binbin
